### PR TITLE
Rubocop 0.47.1

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Updates `rubocop` to `0.47.1`, because I keep getting editor warnings shouted at me since I ran `gem update` the other day and it's annoying 😅.

### CODE REVIEW

File: Library/Homebrew/.rubocop_todo.yml [hunk 1]
```diff
@@ -60,19 +60,26 @@ Lint/ShadowedException:
...
+# Configuration parameters: CountBlocks.
 Metrics/BlockNesting:
   Max: 5
 
-# Offense count: 19
+# Offense count: 21
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 400
+  Max: 364
```
Comment (MikeMcQuaid) [inline Library/Homebrew/.rubocop_todo.yml]: Let's leave this at `400` just so it doesn't 💥 if you need to add a line to e.g. `bottle.rb`

Comment (DomT4) [inline Library/Homebrew/.rubocop_todo.yml]: Was automatically generated but will fix this shortly, no problem.

**Comment by MikeMcQuaid** [comment]
Thanks again @DomT4!

**Comment by DomT4** [comment]
Thanks! My Atom plugin can get off my back every time I edit a `brew` file now 😄.

**Pull request merged**